src/django/api/tiler.py

```diff
@@ -70,11 +67,14 @@ def get_facility_grid_vector_tile(params, layer, z, x, y):
         xmax=xy_bounds.right,
         ymax=xy_bounds.top,
         where_clause=where_clause,
+        limit=HI_LIMIT
     )
 
     with connection.cursor() as cursor:
-        cursor.execute(query, location_params)
+        cursor.execute(query, [location_params, location_params])
```

Comment:
Why do you pass `location_params` twice in the array?

Reply:
Because it's repeated twice in the query. Please find it.

```
            WHERE
                 ST_Contains(ST_Transform(geom, 4326), location)
                 AND ({where_clause})
```

Reply:
Ack.